### PR TITLE
Update jamovi from 2.2.4.0 to 2.2.5.0

### PR DESCRIPTION
**Pull request by neutric**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/jamovi.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url "https://www.jamovi.org/download.html"
-    regex(%r{href=.*?/jamovi[._-]v?(\d+(?:\.\d+)+)[._-]macos\.dmg}i)
+    regex(/.*?jamovi[._-]v?(\d+(?:\.\d+)+)[._-]macos\.dmg/i)
```
Comment (bevanjkay) [inline Casks/jamovi.rb:12]: @neutric Is there a reason this isn't anchored to `href=` anymore?

Comment (neutric) [reply Casks/jamovi.rb:12]: I don't insist on having it gone, I thought this is how @miccal has done it recently.

Comment (neutric) [reply Casks/jamovi.rb:12]: Although now that I look at it again, if you leave it out then `.*?` at the beginning doesn't make much sense.

Comment (bevanjkay) [reply Casks/jamovi.rb:12]: @neutric I think that would only apply when the download link isn't inside a `href` or `data-href` attribute (like a text listing, or JSON/JS).

Comment (neutric) [reply Casks/jamovi.rb:12]: I see! I admittedly felt a little wild when deleting it. ;) Would you mind putting it back, the little insists on me sleeping now.

Comment (neutric) [reply Casks/jamovi.rb]: Thanks!

**Pull request merged**